docs/parse-draft.md

```diff
@@ -0,0 +1,331 @@
+parse
+=====
+
+Use cases we identified:
+
+1. Parse an ISO 8601 string and get the exact information that is mandated by ISO 8601, which means that the whole string is relevant. (24:00 is midnight the following day, :60 seconds are coerced to :59).
+2. Parse something that might look like an ISO 8601 string but might have been generated incorrectly by some other program. In this case it's important to ignore irrelevant parts of the string.
+
+Here are some strings of interest, used in the examples below:
+```javascript
+// Valid string
+valid = '2001-09-08T18:46:40-07:00[America/Vancouver]';
+
+// String where the time affects the date
+time24 = '2020-04-07T24:00:00-07:00[America/Vancouver]';
+
+// Invalid time zone (although valid ISO 8601 string)
+invalidZone = '2001-09-08T18:46:40+01:00[America/Vancouver]';
```

Comment:
This case is particularly interesting, because there are cases where you can't know if the offset is valid or invalid for the time zone (or where you can "know" but be wrong), e.g. `2019-12-15T10:23-02:00[America/Sao_Paulo]` was valid per 2019a tz data (announced March 2019), but not valid per 2019b (announced July 2019) because Brazil stopped observing Daylight Saving Time. Being strict without an escape hatch could also stymie well-intentioned code.

Reply:
I originally thought that the escape hatch would be Temporal.parse. But I agree with you, that is too cumbersome when parsing serialized strings that may have been invalidated by political changes since the time when they were serialized. I'll change this.

Temporal.Absolute would still need to consider both the offset and the IANA name, to be able to disambiguate in the case of a wall-clock time that is repeated in a DST transition.

Reply:
That was the motivation for the disambiguation parameter in the original `Temporal.Absolute.fromString`.

Because there are three valid parsing scenarios:

 1. IANA & Offset have to match (default)
 1. Use the IANA-Zone and only use offset to disambiguate
 1. Use the Offset and ignore IANA

Reply:
Hmm, (3) is not covered in this proposal for how `Temporal.parse()` works ... I'll revise it to make sure the offset is accessible separately.